src/main/java/org/folio/circulation/domain/notice/schedule/DueDateScheduledNoticeHandler.java

```diff
@@ -15,12 +18,23 @@
 import org.folio.circulation.domain.notice.PatronNoticeService;
 import org.folio.circulation.domain.policy.LoanPolicyRepository;
 import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.HttpFailure;
+import org.folio.circulation.support.RecordNotFoundFailure;
 import org.folio.circulation.support.Result;
 import org.joda.time.DateTime;
 
 import io.vertx.core.json.JsonObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class DueDateScheduledNoticeHandler {
+  private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+  private static final String NOTICE_DELETION_MSG = "Deleting scheduled notice {} as referenced {} {} was not found";
+  private static final String USER_REC_TYPE = "user";
```

Comment:
Does the `REC` mean record here? Could it be renamed to `USER_RECORD_TYPE` then?

Reply:
done